python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -331,8 +338,8 @@ def __gt__(self, obj_list):
     def __hash__(self):
         logger.info("hash")
         result = 0
-        for element in self:
-            result ^= hash(element)
+        for element in [hash(l) for l in self._list]:
+            result ^= element
```

Comment:
Why make a temp list, only to then iterate over it? The old code was simpler and probably faster.